locale/de/translation.js

```diff
@@ -249,7 +249,8 @@ export default {
     popupSetColdAddr: 'Gib eine kalte Staking Adresse an', //Set your Cold Staking address
     popupCurrentAddress: 'Aktuelle Adresse', //Current address:
     popupColdStakeNote:
-        'Eine kalte Staking Adresse delegiert Münzen für dich. Sie kann nicht für Transaktionen verwendet werden, so kannst du auch fremde kalte Adressen verwenden!', //A Cold Address stakes coins on your behalf, it cannot spend coins, so it's even safe to use a stranger's Cold Address!
+        'Eine kalte Staking Adresse delegiert Münzen für dich. Sie kann nicht für Transaktionen verwendet werden, so kannst du auch fremde kalte Adressen verwenden!', //A Cold Address stakes coins on your behalf, it cannot spend coins, so it's even safe to use a stranger\'s Cold Address!
+    popupColdStakeOwnerNote: '', //You can set a static Owner Address, or leave it empty to let MPW handle it, <b>do not use a stranger\'s Owner Address!</b>
```

Comment:
also typo here `stranger\'s`

Reply:
It's needed to escape the `'` in JSON (translators keep missing it and breaking the JSON).
And if we switch to `"`, then we have the same issue for quotes.